api/app_analytics/split_testing/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class AppAnalyticsConfig(AppConfig):
+    name = "app_analytics.split_testing"
+    label = "app_analytics_split_testing"
```

Comment:
Ah, of course. I was going off of the feature export apps where I needed a `label` there because of the collision with the other import export module.